packages/gui/src/components/nfts/gallery/NFTGallery.tsx

```diff
@@ -409,9 +425,9 @@ export default function NFTGallery() {
 
   return (
     <LayoutDashboardSub
-      ref={dashboardSub}
       // sidebar={<NFTGallerySidebar onWalletChange={setWalletId} />}
       onScroll={(e: MouseEvent) => {
+        scrollPosition.set('position', (e.target as HTMLElement).scrollTop);
```

Comment:
From the above comment:
`setScrollPosition((e.target as HTMLElement).scrollTop);`